### DISCUSSION TITLE
🖋: Metro BundlerやExpoについての仕組みの説明があったほうが良い？

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#130] [Ideas]
学習コンテンツFBより

- Metro Bundlerが何をしているか図示してほしい
- Metro BundlerやExpoについて図示までしなくてかんたんでいいので仕組みの説明がほしい

**Comment by tani-takanori** [comment]
🍺 さんが資料持っている

**Reply by takei0w0** [reply]
🍺 さんの資料15p,17pを参考にビルドの仕組みを説明する

- タイミング的にGetting Startedのアプリの実行後がよさそう

**Comment by ghost** [comment]
#98 と重複している気がするので、合算すると8票では！？

**Comment by takei0w0** [comment]
#178 にて対応

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
